docs/UserGuide.md

```diff
@@ -281,16 +281,21 @@ Displays a list of upcoming events, ordered by the date, which starts in the nex
 
 Format: `event -u DAYS`
 
+<<<<<<< HEAD
 * The days refer to the number of days after the current date. All events that start within this time frame will be displayed. Events occurring on the day the command is entered will not be displayed.
-* The days **must be a positive integer less than 10000** 1, 2, 3, …​, 9999.
+* The days **must be a positive integer less than 10000** e.g. 1, 2, 3, …​, 9999.
 
 <div markdown="block" class="alert alert-info">
 
 **:information_source: Notes about usage:**<br>
 
-DAYS entered is limited to 9999 days as this sufficiently covers the usage for most NUS students.
+* DAYS entered is limited to 9999 days as this sufficiently covers the usage for most NUS students.
 
 </div>
+=======
```

Comment:
Seems like you accidentally missed out these from the merge conflict. There's a few more below as well!

Reply:
Thanks for the catch! :')